int64 pos_x
int64 pos_y
int64 pos_theta
int64 v_x
int64 v_y
int64 v_theta
int64 bola_x
int64 bola_y